src/modules/vendor/pages/ni/users/trainers/TrainerProfile.vue

```diff
@@ -24,37 +27,26 @@ export default {
   },
   data () {
     return {
-      trainer: {},
       tabsContent: [
         {
           label: 'Infos personnelles',
           name: 'info',
           default: this.defaultTab === 'info',
           component: ProfileInfo,
+          notification: 'profiles',
         },
       ],
     }
   },
   async mounted () {
-    await this.refreshTrainer();
+    await this.$store.dispatch('rh/getUserProfile', { userId: this.trainerId });
   },
   computed: {
     userProfile () {
       return this.$store.getters['rh/getUserProfile'];
     },
     userIdentity () {
-      if (this.userProfile) return formatIdentity(this.userProfile.identity, 'FL');
-      return '';
-    },
-  },
-  methods: {
-    async refreshTrainer () {
-      try {
-        await this.$store.dispatch('rh/getUserProfile', { userId: this.trainerId });
-      } catch (e) {
-        console.error(e);
-        this.trainer = {};
-      }
```

Comment:
pourquoi avoir enlever cette methode ?
par contre tu peux enlever le this.trainer = {};

Reply:
Parce qu'elle n'etait utilisee que dans mounted, du coup ca ne me semblait pas utile.